deploy/helm/sumologic/values.yaml

```diff
@@ -30,22 +30,43 @@ eventsDeployment:
 
 sumologic:
   ## Setup
-
-  # If enabled, a pre-install hook will create Collector and Sources in Sumo Logic
-  setupEnabled: true
-
-  # If enabled, accessId and accessKey will be sourced from Secret Name given
-  # envFromSecret: accessCredentials
-
-  # Sumo access ID
-  #accessId: ""
-
-  # Sumo access key
-  #accessKey: ""
-
-  # Sumo API endpoint
-  # ref: https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
-  #endpoint: ""
+  setup:
+    clusterRole:
+      annotations:
+        helm.sh/hook: pre-install
+        helm.sh/hook-weight: 1
+        helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    clusterRoleBinding:
+      annotations:
+        helm.sh/hook: pre-install
+        helm.sh/hook-weight: 2
+        helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    configMap:
+      annotations:
+        helm.sh/hook: pre-install
+        helm.sh/hook-weight: 2
+        helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    job:
+      #envFromSecret: sumo-api-secret
+
+      # Sumo access ID
+      #accessId: ""
```

Comment:
changing these will break automation passing `--set sumologic.accessId=x` etc. It would also require a docs update.

Would prefer we keep these auth variables as they were before.

Reply:
Sounds good